plugins/kernel_v2/config/vm_provisioner.rb

```diff
@@ -105,6 +112,9 @@ def add_config(**options, &block)
       def finalize!
         return if invalid?
 
+        # Force communicator to be true or false based on turthy-ness
+        @communicator_required = @communicator_required ? true : false
```

Comment:
Similar to my comment on a different PR, it would be good to have a validation step for this config option in case someone sets it to a string or something other than true/false